src/Security/Authorization.cs

```diff
@@ -140,7 +140,14 @@ public unsafe class Authorization : INativeObject, IDisposable {
 		[DllImport (Constants.SecurityLibrary)]
 		extern static int /* OSStatus = int */ AuthorizationCreate (AuthorizationItemSet *rights, AuthorizationItemSet *environment, AuthorizationFlags flags, out IntPtr auth);
 
-		[Deprecated (PlatformName.MacOSX, 10,7)]
+#if !NET
+		[Deprecated (PlatformName.MacOSX, 10, 7)]
+#else
+		[UnsupportedOSPlatform ("macos10.7")]
+#if MONOMAC
+		[Obsolete ("Starting with macos10.7 ", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
```

Comment:
if there was no message then an `[Obsolete]` is not needed, since it gives no additional clue to the developer. The compiler warnings will already tell the developer not to use the API for that OS version.

IOW the new dotnet attributes don't give us the flexibility of adding a custom message. That's why we're using `[Obsolete]` since most of our `[Obsoleted]` (or `[Deprecated]` in this case) comes with a replacement or additional advice to developers.

Reply:
The documentation states the following: 
```
Use a launchd-launched helper tool and/or the Service Management framework for this functionality.
```